### ISSUE TITLE
[BITV]: Adapt tooltips on NcAvatar for files_sharing

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#36357]
## Problem

Custom tooltips were already exchanged by native tooltips. Though some small structure should be adapted. Now the usage of `aria-label` and `title` is not needed anymore. It is possible just to use `display-name` attribute as before.

## Solution

Remove `title` and `aria-label` attributes from NcAvatar components in files_sharing.
